core/client-entry.ts

```diff
@@ -233,66 +233,37 @@ const invokeClientEntry = async () => {
   EventBus.$on('sync/PROCESS_QUEUE', data => {
     if (typeof navigator !== 'undefined' && navigator.onLine) {
       // event.data.config - configuration, endpoints etc
-      const storeView = currentStoreView()
-      const dbNamePrefix = storeView.storeCode ? storeView.storeCode + '-' : ''
       const syncTaskCollection = Vue.prototype.$db.syncTaskCollection
+      const currentUserToken = rootStore.getters['user/getUserToken']
```

Comment:
this should  be completely moved to user module  imho

Reply:
OK, I moved it to the module